BarrysPeanutsAkkaJava/readme.md

```diff
@@ -0,0 +1,54 @@
+# Implementing a flattened version of the Actor model under Akka
+
+This project demonstrates implementing a Temporal Workflow using only using signals. Taking a "signaling only" 
```

Comment:
That comment seems out of place. That's about Temporal and signals. This file is about Akka.

Reply:
addressed